lib/schema/webContentSchema2.json

```diff
@@ -0,0 +1,98 @@
+{
+    $schema: "http://json-schema.org/schema#",
```

Comment:
Again this can't be a valid JSON file; the attributes are missing double-quotes.